docs/ecs_guide.md

```diff
@@ -182,22 +182,14 @@ To create an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_role
         "containerDefinitions": [
             {
                 "name": "prefect-worker",
-                "image": "prefecthq/prefect",
+                "image": "prefecthq/prefect:2-latest",
                 "cpu": 512,
                 "memory": 1024,
                 "essential": true,
                 "command": [
-                    "pip",
-                    "install",
-                    "prefect-aws",
-                    "&&",
-                    "prefect",
-                    "worker",
-                    "start",
-                    "--pool",
-                    "my-ecs-pool",
-                    "--type",
-                    "ecs"
+                  "/bin/sh",
+                  "-c",
+                  "pip install prefect-aws && prefect worker start --pool my-ecs-pool --type ecs"
```

Comment:
my instinct is that we should stick with the original "exec" form here instead of `/bin/bash -c` ("shell") - I feel like there could be weirdness associated with signal / error reporting - thoughts?

Reply:
I'd agree generally but I tried quite a few variations on this based upon the existing content of the guide and couldn't get it to work. Logs from the worker container looked like it was interpreting each part of the command separately.

Reply:
ah interesting! okay - yeah i think my comment was more philosophical than practical so if that's what's working then I defer to you